### PR TITLE
Drew MacNicholas

### PR DESCRIPTION
**Pull request by dmacnicholas**
Ran out of time near end of iteration 3

### CODE REVIEW

**Comment by dmacnicholas** [comment]
Ran out of time near end of iteration 3

File: lib/book.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  def author
+    @author_first_name + " " + @author_last_name
+  end
+
+  def publication_year
+    @publication_date[-4..-1]
+  end
+end
```
Comment (megstang) [inline lib/book.rb:19]: This looks good, but you do not need attr readers for author_first_name, author_last_name, or publication_date. If you don't need to get those attributes outside of the class by calling a method, then you do not need the attr readers.

File: lib/author.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class Author
+  attr_reader :first_name, :last_name, :books
```
Comment (megstang) [inline lib/author.rb:2]: same thing here.. no need for first name or last name. We never need to call .first_name or .last_name, so for that reason, we don't need these attr_Readers.

File: lib/author.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  def write(title, publication_date)
+    new_book = Book.new({
+      author_first_name: @first_name,
+      author_last_name: @last_name,
+      title: title,
+      publication_date: publication_date})
+    @books << new_book
+    new_book
+  end
```
Comment (megstang) [inline lib/author.rb:22]: nice job!

**Review by megstang — COMMENTED** [review]
Nice work! I'll follow up via slack later today with scores.

**Pull request closed without merging**